docs/overview.mdx

```diff
@@ -37,6 +37,64 @@ Casdoor serves both the web UI and the login requests from the application users
 
 9. Casdoor supports mainstream databases: MySQL, PostgreSQL, SQL Server etc,  and support the extension of new database with plugins.
 
+## How it works:
+
+### Step 0 (Pre-knowledge)
+1. `Casdoor`'s authorization process is built upon the OAuth 2.0 protocol, therefore it's highly recommended to take a brief look at how indeed does OAuth 2.0 works. An [introduction](https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2) of OAuth 2.0.
+
+![image](https://user-images.githubusercontent.com/46557895/132336955-e25ab988-64b3-4087-b332-d9e724322acd.png)
+
+### Step 1 (Authorization Request) 
+
+Your `Application` (could be a website or whatever) should compose an `URL` in this format `endpoint/login/oauth/authorize?client_id=xxx&response_type=code&redirect_uri=xxx&scope=read&state=xxx`. In the `URL` replace `endpoint` with your `Casdoor`'s host URL, and replace `xxx` with your own info.
+
+**Hints**:  
+How to fill out the `xxx` parts?  
+
```

Comment:
To add a hint, I think use the Docusaurus admonition might be better

Reply:
will do

Reply:
Please note the admonitions need a carriage return

Like this:

:::tip

Hello

:::